pynecone/state.py

```diff
@@ -117,6 +117,12 @@ def __init_subclass__(cls, **kwargs):
         if parent_state is not None:
             cls.inherited_vars = parent_state.vars
 
+        cls.backend_vars = {
```

Comment:
Don't we need to add this as a Pydantic field to the `pc.State` class?

```python
class State(Base, ABC):
    """The state of the app."""
    ...
    # Vars inherited by the parent state.
    backend_vars: ClassVar[Dict[str, Var]] = {}
```

Reply:
I didn't notice any problem without the declaration, but I agree we should add it.

However, `Dict[str, Var]` => `Dict[str, Any]` (because backend variables are never transformed into Var, they are simply pure Python variables.
I'm not sure what the `ClassVar` does, but if it's needed I can add it.

Reply:
ClassVar just makes it so it's not tied to the instance but the class instead. So it won't appear in the `.dict` and `.json` methods I believe.